.ci/test_r_package.sh

```diff
@@ -167,45 +167,43 @@ elif [[ $R_BUILD_TYPE == "cran" ]]; then
     if [[ "${TASK}" == "r-rchk" ]]; then
         echo "Checking R package with rchk"
         mkdir -p packages
-        cp ${PKG_TARBALL} packages
+        cp "${PKG_TARBALL}" packages
         RCHK_LOG_FILE="rchk-logs.txt"
         docker run \
-            -v $(pwd)/packages:/rchk/packages \
+            -v "$(pwd)"/packages:/rchk/packages \
```

Comment:
```suggestion
            -v "$(pwd)/packages":/rchk/packages \
```

The full path, please.